app/pages/index.vue

```diff
@@ -64,7 +64,7 @@ import { Component, Vue } from 'vue-property-decorator'
 @Component({
   components: {}
 })
-export default class Home extends Vue {}
+export default class index extends Vue {}
```

Comment:
一応慣例的にはクラス名は大文字で始めるパターンが多いんだけど、そもそもこの場合はクラス名を明示的に付ける必要性があまりないから、こんな感じでどうだろう🐱？

```suggestion
export default class extends Vue {}
```

Reply:
@keitakn うん！クラス名を宣言する必要はないね！
提案してもらった内容で修正したよ！